docs/wxm-devices/wifi-m5-bundle/m5-troubleshooting.mdx

```diff
@@ -184,15 +184,15 @@ In case you desire to change the internal/external LoRa antenna, follow the foll
    Remove the cable, which is connected to a specific reception on the green board and
    connect the desired cable to the same reception.
 1. Clasp the caps and the cap back and screw the device.
-1. In case you changed to external LoRa antenna, connect the external antenna to the corresponding port on the side of the WG1000 Gateway.
+1. In case you changed to external RF antenna, connect the external antenna to the corresponding port on the side of the WG1000 Gateway (LoRa cap).
 
 ##### Internal parts of the WG1000 Gateway
 
 a. Bottom is the first cap you remove once the WG1000 Gateway is unscrewed.
 
 b. The second cap in grey is the GPS
 
-c. The third cap in blue is the LoRa
+c. The third cap in blue is the LoRa (RF Connection for the weather station)
```

Comment:
Based on the description of the task we don't use LoRa on the M5 so I suggest replacing it with:

> The third cap in blue is the RF Connection to the weather station

Reply:
same as the above